components/org.wso2.carbon.identity.api.server.webhook.management/org.wso2.carbon.identity.api.server.webhook.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/webhook/management/v1/model/WebhookResponse.java

```diff
@@ -253,14 +253,14 @@ public boolean equals(java.lang.Object o) {
             Objects.equals(this.updatedAt, webhookResponse.updatedAt) &&
             Objects.equals(this.endpoint, webhookResponse.endpoint) &&
             Objects.equals(this.eventSchema, webhookResponse.eventSchema) &&
-            Objects.equals(this.description, webhookResponse.description) &&
+            Objects.equals(this.name, webhookResponse.name) &&
             Objects.equals(this.eventsSubscribed, webhookResponse.eventsSubscribed) &&
```

Comment:
Shall we change eventsSubscribed to channelsSubscribed or subscribedChannels

Reply:
fixed with https://github.com/wso2/identity-api-server/pull/902/commits/0386f0e0fc36edb1fa5b62bd5ea86eb2765b3487